src/ProjectOrigin.WalletSystem.Server/ViewModels/ClaimViewModel.cs

```diff
@@ -7,7 +7,7 @@ namespace ProjectOrigin.WalletSystem.Server.ViewModels;
 
 public record ClaimViewModel
 {
-    public required Guid Id { get; init; }
+    public required Guid ClaimId { get; init; }
```

Comment:
Wouldn't it be better to make use of strong types, to avoid this issue from occurring again?

Define a readonly record struct for it like this perhaps?:

```
public readonly record struct ClaimId(Guid Value)
{
    public static ClaimId NewClaimId() => new ClaimId(Guid.NewGuid());
    public static ClaimId Empty => new ClaimId(Guid.Empty);
```